docs/batch-changes/configuring-credentials.mdx

```diff
@@ -209,6 +211,12 @@ Adding a GitHub app is done through the Batch Changes section of your user setti
 
 Code hosts with GitHub apps configured are indicated by a green tick to the left of the code host name as well as details about the GitHub app on the right, while code hosts without credentials have an empty red circle next to them.
 
+### Updating Installations and Permissions for a GitHub App
+
+Currently, Sourcegraph doesn't handle changed installations or permissions for a GitHub app.
+
+To update those properties, you can delete the current credential and create a new one.
+
```

Comment:
```suggestion
### Updating Installations and Permissions for a GitHub App

Once a GitHub app is installed in Sourcegraph, if you need to update the permissions in the GitHub App, you'll need to delete the current credential and create a new one.

```
Not sure I'm using "installed" correctly here...

Reply:
Looks good to me!